frontend/packages/knative-plugin/src/components/overview/__mocks__/overview-knative-mock.ts

```diff
@@ -27,3 +27,19 @@ export const mockRoutesData = [
     },
   },
 ];
+
+export const mockConfigurationsData = [
+  {
+    apiVersion: 'serving.knative.dev/v1beta1',
+    kind: 'Configuration',
+    metadata: {
+      name: 'nodejs-ex-git',
+      uid: 'c09c5883-c3ce-445c-a181-42e152578121',
+      namespace: 'ari-test',
```

Comment:
nit: it'll be good to update `namespace` as common in this file i.e `ari-test` and `jai-test` to `serv-app-test` or something similar.

Reply:
Applies to metadata name as well i.e let's use the same as we have in `mockRoutesData`